cmake/libvpx.cmake

```diff
@@ -59,6 +93,8 @@ else()
         list(APPEND include_directories
             ${libvpx_loc}/source/config/linux/arm-neon
         )
+        set(ads2gas ${libvpx_loc}/source/libvpx/build/make/ads2gas.pl)
```

Comment:
why is this needed? why just don't specify the path inside target_ads2gas_sources?

Reply:
That's because there's actually more variants of the script, like e.g. for 32-bit Windows or Apple Devices; we don't currently use any of those.

Another reason is to allow the check for Perl to be conditional

Reply:
It feels too hackish to me, why don't do the check for the system inside the function?

You can also move find_package inside the function, look here: https://github.com/telegramdesktop/tdesktop/blob/dev/Telegram/cmake/generate_scheme.cmake

Reply:
Thanks, that's a good idea, will do!